package/firmware/omnia-mcu-firmware/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2023 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-mcu-firmware
+PKG_DISTNAME:=omnia_hw_ctrl
+PKG_VERSION:=3.4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_SUBDIR:=$(PKG_DISTNAME)-binaries-v$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.bz2
```

Comment:
I am thinking. Do we really need to have extra variable - ``PKG_DISTNAME``? Can not we have it only as ``omnia-hw-ctrl``?

Reply:
I was inspired by package `u-boot-envtools`... Notice that the variable is then used at 3 places.